CavalcanteLucas/todolist
# Feature/OL40-todo_create

**Link to task:**
https://labcodes.atlassian.net/browse/OL-40

**How to test:**
- With a user logged in:
i) press the button to create a To-Do List on the top of left bar;
ii) fill the form to define the List's title and press 'Save';
iii) Check if success message appears when operation is complete;
-  After logout, login with another user. Check if the just created List is *not* available, ensure lists have correct owners. 
- Check if standard view for 404 requests is rendered when accessing any invalid route.

**Description of your solution:**
This PR enables To-Do lists to be stored on the database via API through a React interface. One can now create To-Do Lists from the user's dashboard. An 'Action Bar' at the left side of the user's dashboard is introduced in this PR. It contains a button that opens a modal with a form asking to the new To-Do List's title. To-Do lists created by one user should only be visible by him. Messages of success / failure from interaction with DB should be visible at the top of dashboard. This PR also ensures all images used in the site are locally available. Additionally, this PR establish a standard view for 404 requests.
